### ISSUE TITLE
In a demo version, what do we keep from the production database?

### ISSUE THREAD

**Issue by hmpf** [hmpf/easydmp#110]
I'm thinking:

Keep:

* All published templates
* All eestore stuff

Drop/trunc:

* Eventlog
* Plans

But what about users? Keep is_staff/is_superuser?

Clean during dump or after restore? Both?

**Comment by hmpf** [comment]
By splitting up schema and data, we can avoid dumping what we don't want, like eventlog/plans. That won't work for getting a subset of templates and a subset of users.

**Comment by hmpf** [comment]
Existing tools: 
* (Jailer)[https://github.com/Wisser/Jailer]
* (Condenser)[https://github.com/TonicAI/condenser]

**Comment by adilhasan** [comment]
yes no users, keep the super users. Don't keep the plans. Do keep the templates and eestore stuff

We should ideally cleanup regularly the plans.

**Comment by adilhasan** [comment]
I don't think we should cleanup accounts.

**Comment by adilhasan** [comment]
Do you mean accounts that are created in the demo? Yes, I agree with you.
